hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMDirectoryCreateRequest.java

```diff
@@ -149,6 +150,12 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     List<OmKeyInfo> missingParentInfos;
 
     try {
+      ResolvedBucket bucket = ozoneManager.resolveBucketLink(keyArgs);
+      keyArgs = bucket.update(keyArgs);
```

Comment:
And also do you think it would be useful to log actual bucket/volume request, and then resolvedBucket/resolvedVolume instead of just updating volumeName/bucketName directly.

Reply:
Audit log includes both.